docs/assets/diagrams/linking/06-sign-challenge.puml

```diff
@@ -0,0 +1,125 @@
+@startuml
+
+title: Sign Challenge
+
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+'start flow
+
+App -> Server ++: ""**PUT /consents/{ID}**""\n\
```

Comment:
I do wonder if the server should be exposing a different looking API to the application here - the server could simplify a lot of what the App needs to keep a track of regarding the `Consent` object.

Reply:
updated the flow and removed this part